meteor-app/imports/ui/components/textmenu.js

```diff
@@ -0,0 +1,104 @@
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+import Menu from 'material-ui/Menu';
+import Popover from 'material-ui/Popover';
+import Divider from 'material-ui/Divider';
+import React from 'react';
+import _ from 'lodash';
+
+export default class TextMenu extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { open: [] };
+    this.data = this.props.data;
```

Comment:
Why do you need this extra reference to the data passed in by props? Keep in mind that props can change and the constructor is called only once per instance.